.travis.yml

```diff
@@ -1,29 +1,16 @@
-language: node_js
-node_js:
-  - "6"
+sudo: required
 
-# This (sudo: false) is needed to "run on container-based infrastructure" on
-# which cache: is available
-# http://docs.travis-ci.com/user/workers/container-based-infrastructure/
-sudo: false
+language: python
 
-before_install:
-  - pip install --user restructuredtext_lint
-  - pip install --user flake8 pycodestyle pep8-naming
-  - flake8 -v .
-  - rst-lint README.rst
+python:
+  - "2.7"
+
+services:
+  - docker
 
-before_script:
-  - npm install -g grunt-cli
+before_install:
+  - git clone --depth=50 --branch=web-apps git://github.com/jburel/omero-test-infra .omero
 
 script:
-  - python setup.py sdist install --user || exit 1
-  - if [ ! -f "omero_figure/static/figure/templates.js" ]; then exit 1 ;fi
-  - if [ ! -f "omero_figure/static/figure/figure.js" ]; then exit 1 ;fi
-  # Test demo build
-  - grunt demo || exit 1
-  - if [ ! -f "demo/figure.js" ]; then exit 1 ;fi
-  - if [ ! -f "demo/index.html" ]; then exit 1 ;fi
-  - if [ ! -f "demo/omero_figure/static/figure/templates.js" ]; then exit 1 ;fi
-  - if [ ! -f "demo/omero_figure/static/figure/css/figure.css" ]; then exit 1 ;fi
-  - if [ ! -f "demo/omero_figure/static/figure/images/luts_10.png" ]; then exit 1 ;fi
+  - APP_BUILD=false .omero/app-docker
+  - SET_UP=false DIR_SCRIPTS=omero_figure/scripts/omero .omero/scripts-docker
```

Comment:
With https://github.com/openmicroscopy/omero-test-infra/pull/3#issuecomment-362535909, I don't think `APP_BUILD` will be needed, but interested to hear if you think so. (I _think_ it will work anyway)

In general, I'd be for trying to reduce the number of parameters (i.e. environment variables) that we pass to `.omero/(*-)docker` in favor having the `.omeroci/` scripts override as necessary.